Client/Frontend/Browser/OpenInHelper.swift

```diff
@@ -21,8 +21,13 @@ struct MIMEType {
     static let PlainText = "text/plain"
     static let PNG = "image/png"
     static let WebP = "image/webp"
+    static let xHTML = "application/xhtml+xml"
+    
+    static func kindOfHTML(_ mime: String?) -> Bool {
```

Comment:
Do you think `static` is cleaner here than calling:

`response.mimeType?.isHTML == false`

Reply:
Nope, changing it to instance type. Thanks :)